app/services/follow_service.rb

```diff
@@ -39,7 +39,9 @@ def call(source_account, target_account, options = {})
     if (@target_account.locked? && !@options[:bypass_locked]) || @source_account.silenced? || @target_account.activitypub?
       request_follow!
     elsif @target_account.local?
-      direct_follow!
+      follow = direct_follow!
+      trigger_webhooks follow.id
```

Comment:
Looks like `direct_follow!` already has some calls to other workers ... maybe just add there (avoids the local var and explicit return up here)

Reply:
Didn't see that. This is better indeed. Also makes the `trigger_webhooks` function obsolete.